File: app/src/main/java/com/alex/materialdiary/sys/common/models/diary_day/HomeworkPrevious.java
package com.alex.materialdiary.sys.common.models.diary_day;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class HomeworkPrevious {

    @SerializedName("DATE")
    @Expose
    private String date;
    @SerializedName("HOMEWORK")
    @Expose
    private String homework;

    public String getDate() {
        return date;
    }

    public void setDate(String date) {
        this.date = date;
    }

    public String getHomework() {
        return homework;
    }

    public void setHomework(String homework) {
        this.homework = homework;
    }

}

File: app/src/main/java/com/alex/materialdiary/sys/common/models/diary_day/HomeworkPrevious.kt
package com.alex.materialdiary.sys.common.models.diary_day

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import com.google.gson.annotations.Expose
import kotlinx.parcelize.Parcelize

@Parcelize
class HomeworkPrevious : Parcelable {
    @SerializedName("DATE")
    @Expose
    var date: String? = null

    @SerializedName("HOMEWORK")
    @Expose
    var homework: String? = null
}